app/views/users/index.html.erb

```diff
@@ -1 +1,22 @@
-<h1>Here is a list of users</h1>
\ No newline at end of file
+<%= stylesheet_link_tag ('home') %>
+
+<section class="users-home">
+  <% @users.each do |user| %>
```

Comment:
- [ ] [OPTIONAL] You can use `app/views/users/_user.html.erb` instead of `objects.each` for better practices, 
e.g
```ruby
    # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```